src/tests/test_apriori.cpp

```diff
@@ -39,6 +39,26 @@ void CheckAssociationRulesListsEquality(
     SUCCEED();
 }
 
+void CheckSupportAndConfidence(std::list<model::ARStrings> const& actual,
+                               std::set<std::string> const& lhs, std::set<std::string> const& rhs,
+                               double expected_support, double expected_confidence) {
+    for (auto const& rule : actual) {
+        std::set<std::string> actual_lhs(rule.left.begin(), rule.left.end());
+        std::set<std::string> actual_rhs(rule.right.begin(), rule.right.end());
+
+        if (lhs == actual_lhs && rhs == actual_rhs) {
+            ASSERT_DOUBLE_EQ(rule.support, expected_support)
+                    << "supports don't match: expected " << expected_support
+                    << ", got: " << rule.support;
+            ASSERT_DOUBLE_EQ(rule.confidence, expected_confidence)
+                    << "confidences don't match: expected " << expected_confidence
+                    << ", got: " << rule.confidence;
```

Comment:
If I remember correctly Assert will make test fail immedeatly. So you'll never get an output from the second one. I guess u can change them to Expect and than make it fail manually by Fail instead of return